README.md

```diff
@@ -2,24 +2,134 @@
     <img src="./docs/media/dynamiqs_logo.png" width="520" alt="dynamiqs library logo">
 </h1>
 
-Quantum systems simulation with PyTorch.
+[![ci](https://github.com/dynamiqs/dynamiqs/actions/workflows/ci.yml/badge.svg)](https://github.com/dynamiqs/dynamiqs/actions/workflows/ci.yml)  ![python version](https://img.shields.io/badge/python-3.8%2B-blue) [![license: GPLv3](https://img.shields.io/badge/license-GPLv3-yellow)](./LICENSE) [![code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-This library provides differentiable solvers for the Schrödinger Equation, the Lindblad Master Equation and the Stochastic Master Equation. All the solvers are implemented using PyTorch and can run on GPUs.
+<!-- PyTorch implementation of differentiable quantum dynamics solvers. -->
+<!-- Differentiable Quantum Solvers -->
+
+High-performance quantum systems simulation with PyTorch.
+
+The **dynamiqs** library enables simulation of large quantum systems on GPUs, and the computation of the gradient of any function depending on the evolved quantum state. We provide differentiable solvers for the Schrödinger Equation, the Lindblad Master Equation, and the Stochastic Master Equation. All of these solvers are implemented using PyTorch and can run on either a CPU or a GPU.
 
 :hammer_and_wrench: This library is under active development and while the APIs and solvers are still finding their footing, we're working hard to make it worth the wait. Check back soon for the grand opening!
 
+Here are the main features we're proud of:
+
+- Run your simulation on GPUs, to gain a significant speedup for large Hilbert space dimensions.
+- Batch your simulations to run them concurrently on multiple different Hamiltonians or initial states.
+- Explore quantum-specific solvers that preserve the properties of the state, such as positivity and trace.
+- Compute the gradient of any function that depends on the quantum state at any given time with respect to any parameter of the Hamiltonian, Lindblad jump operators, or initial state.
+- Choose to compute the gradient using fast automatic differentiation for smaller systems, or opt for the adjoint method for constant memory cost.
+- Use the library as a drop-in replacement for [QuTiP](https://qutip.org/) by directly passing QuTiP-defined quantum objects to our solvers.
+- Implement your own solvers with ease by subclassing our base solver class and focusing directly on the solver logic.
+- Enjoy reading our carefully crafted documentation on our website: <https://www.dynamiqs.org>.
+- Benefit from a well-structured library with a rigorous test suite, continuous integration and best coding practices (and a team of friendly enthusiast developers :nerd_face:).
```

Comment:
```suggestion
```
These ones are not so much related to the library, but rather they should be automatic for any well-written library. So I would keep them out. This makes it so readers can focus on the essentials a bit more also by having less points.